packages/core/admin/admin/src/content-manager/pages/ListView/index.js

```diff
@@ -623,6 +623,19 @@ function ListView({
                             );
                           }
 
+                          if (typeof cellFormatter === 'function') {
+                            return (
+                              <Td key={key}>
+                                {cellFormatter(rowData, {
+                                  key,
+                                  name,
+                                  formatMessage,
+                                  ...rest,
+                                })}
+                              </Td>
+                            );
+                          }
```

Comment:
I'm not really a fan of the `cellFormatter` pattern. Couldn't I just import [the rw component](https://github.com/strapi/strapi/blob/c319af46020ee500bc5f96796ec8fa79ab73c696/packages/core/admin/ee/admin/content-manager/components/ListViewTable/CellContent/ReviewWorkflowsStage/ReviewWorkflowsStageEE.js#L1) directly in the list view instead of having the cellFormatter on the headers object and then calling it here? How do y'all feel about that? Maybe there is a case where we still need the cellFormatter?

Reply:
I'm not a fan of that api either, i'd have to defer to @gu-stav for some direction though as i'm not _super_ familiar with the work here (and it also being EE 🙃 )

Reply:
I'm not a fan either, but since this pattern is currently used in many places I've kept it in sync. Could you give me some guidance in https://github.com/strapi/strapi/pull/17047 how you would refactor it?

Happy to change it :)